src/rust/engine/src/externs/interface.rs

```diff
@@ -985,6 +985,22 @@ async fn workunit_to_py_value(
             ));
         }
 
+        if let Some(command_digest) = metadata.command {
+            artifact_entries.push((
```

Comment:
This exposes them to python in the artifacts dictionary, like `work_unit["artifacts"]["command_digest"]` (see the test) I put them in artifacts simply because stdout and stderr where already there. Not tied to this location if you want me to move them though

Reply:
Seems fine to me!